premake5.lua

```diff
@@ -79,7 +79,9 @@ filter("configurations:Release")
     "NDEBUG",
     "_NO_DEBUG_HEAP=1",
   })
-  optimize("On")
+  optimize("speed")
+  inlining("Auto")
+  floatingpoint("Fast")
```

Comment:
I think you mentioned side effects to this change, or was that for AVX2? (which isn't in this PR?)

Reply:
It was only for AVX2 and it was limited to only a few sections of games. Blue Dragon being the most noticeable and it only happens sometimes. So not sure if it's actually a problem somewhere else that just shows up with AVX2.